### ISSUE TITLE
#710 broke StdEventMonitor on Linux/CentOS

### ISSUE THREAD

**Issue by msuljic** [eudaq/eudaq#712]
#710 while appeared to resolve compilation issues on MacOS, broke StdEventMonitor on CentOS.

Command
```
$ StdEventMonitor -d run497194033_221211201609.raw -l DEBUG -e 100 -E 1000
```
yields error messages like this for every event:
```
[2023-04-12 15:51:01.630] (WARN) [unknown sender]WARNING, no StdEventConverter for RawEvent with ExtendWord(1605811347) and Description(ITS3global)
Factory<0x7fe4dd2df000>:  Unknown class ID: <1605811347>
```
Reverting to latest commit before the PR solves the issue on CentOS (ofc, compilation on MacOS will likely revert to be broken).

Additional info - build commands:
```
$ sh make_clean.sh && cd build
$ cmake -DEUDAQ_BUILD_STDEVENT_MONITOR=ON -DUSER_ITS3_BUILD=ON -DEUDAQ_BUILD_PYTHON=ON ..
$ make -j 10 && make install
```

**Comment by simonspa** [comment]
Hi @msuljic 

thanks for reporting this - and sorry @ffeindt that this caused you to dig around why your converters don't work anymore.

I investigated what the reason is and how we could solve it. Essentially the eudaq build system here *abuses* a variable that CMake uses to add prefixes (like `lib`) to adjust library names to the host system. Now EUDAQ just overwrites it to add its `eudaq_` prefix...

My approach that would not involve reworking large fractions of the build system to properly add the prefix to the target names instead of to the library prefix would be this: https://github.com/eudaq/eudaq/pull/714

Cheers,
Simon

**Comment by stephanlachnit** [comment]
Yeah I can confirm #710 breaks everything... Can we revert until we have a proper fix?

**Comment by simonspa** [comment]
@stephanlachnit care to fix? :)
